test-data/unit/fixtures/async_await.pyi

```diff
@@ -11,7 +11,7 @@ class int: pass
 class str: pass
 class dict: pass
```

Comment:
Maybe you could also fix ``dict`` and ``set`` (on next line)? There are few of them.